velox/exec/tests/TableWriteTest.cpp

```diff
@@ -610,3 +562,122 @@ TEST_F(TableWriteTest, writeNoFile) {
   execute(plan, std::make_shared<core::QueryCtx>(executor_.get()));
   ASSERT_TRUE(fs::is_empty(outputDirectory->path));
 }
+
+TEST_F(TableWriteTest, insertUnpartitionedTableWithExistingTableType) {
```

Comment:
Is this testing appending data to an existing unpartitioned table? If so, perhaps, rename to appendToUnpartitionedTable.